select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2ResourcesBehavior.java

```diff
@@ -60,20 +56,5 @@ public void renderHead(Component component, IHeaderResponse response)
 		{
 			response.render(CssHeaderItem.forReference(settings.getCssReference()));
 		}
-		//i18n
-		try
-		{
-			String name = String.format("res/js/i18n/%s.js", Session.get().getLocale().toLanguageTag());
-			URL lang = getClass().getResource(name);
-			if (lang != null)
```

Comment:
Ok, thinking again I got to the point that I might be a good idea to have such a check, so I added one (see latest commit).